packages/tools/examples/ultrasoundAnnotation/index.ts

```diff
@@ -0,0 +1,277 @@
+import type { Types } from '@cornerstonejs/core';
+import { RenderingEngine, Enums } from '@cornerstonejs/core';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import WindowLevelTool from '../../src/tools/WindowLevelTool';
+
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  StackScrollTool,
+  ZoomTool,
+  UltrasoundAnnotationTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { ViewportType } = Enums;
+const { MouseBindings } = csToolsEnums;
+
+const toolGroupId = 'STACK_TOOL_GROUP_ID';
+const leftClickTools = [UltrasoundAnnotationTool.toolName];
+const defaultLeftClickTool = leftClickTools[0];
+const currentLeftClickTool = leftClickTools[0];
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Ultrasound annotations',
+  'Showcases how the bline and pleura annotations of ultrasound images are displayed and rendered correctly'
```

Comment:
Let's clarify in the description what kind of metadata this tool needs. This will prevent people from assuming it doesn't require any.